app/views/content_items/publication.html.erb

```diff
@@ -29,7 +29,27 @@
   <div class="govuk-grid-column-two-thirds responsive-bottom-margin">
     <%= render 'components/important-metadata', items: @content_item.important_metadata %>
     <div class="responsive-bottom-margin">
-       <%= render 'publication_inline_body' %>
+      <%= render 'govuk_publishing_components/components/heading',
+        text: t("publication.documents", count: 5), # This should always be pluralised.
```

Comment:
Maybe in another commit after the first commit, you could change this. The `count: 5` feels weirdly arbitrary. Even `2` would be clearer!

Reply:
It's tempting, but that means going through all 61 locale files and working out what to do with it. There's definitely a bigger problem with the use of I18n in this app, so I'd like to leave it for someone else to tackle.